### DISCUSSION TITLE
WSA 2304 Update!

### DISCUSSION THREAD

**Discussion by hamza-usmani** [microsoft/WSA#293] [Announcements]
We've shipped an update for Windows Subsystem for Android™ on Windows 11 to the [Windows Subsystem for Android™ Preview Program](https://learn.microsoft.com/en-us/windows/android/wsa/preview-program). This update (2304.40000.3.0) will add package verification, improved reliability and AppLink support.

# What’s New
 
- **Package verification** for apps on WSA: Android apps are scanned using anti-virus software installed on Windows prior to app installation.
- Ability for users to configure how much memory to assign to Android
- Android apps will be launched when a user opens the supported app link from any app (**Android AppLink support**)
- Linux kernel updated to 5.15.94 
- WSA reliability and performance improvements
 
# Giving feedback

If you are having issues with Windows Subsystem for Android™ – please file feedback via Feedback Hub under Apps > Windows Subsystem for Android™. For more information about troubleshooting and submitting feedback: [Troubleshooting and FAQ for mobile apps on Windows](https://support.microsoft.com/en-us/windows/troubleshooting-and-faq-for-mobile-apps-on-windows-d6062afd-98a1-4018-a7c8-6b3b680a2ea5).

Thanks,
Windows Subsystem for Android™ Team

**Comment by HotCakeX** [comment]
Awesome update and feature list 👏👏

Screenshots

![image](https://user-images.githubusercontent.com/118815227/236602377-39ffc031-b0f2-4d35-8f3e-c41f2348c3bf.png)
![image](https://user-images.githubusercontent.com/118815227/236602380-bcd570bf-7879-420f-8583-a7e8ae7abba7.png)
![image](https://user-images.githubusercontent.com/118815227/236602383-a63fd73a-164c-4b4c-85b9-4b905ceee5ee.png)

**Comment by riverar** [comment]
@hamza-usmani Can you provide more detail around _Package verification for apps on WSA_? Is the WSA filesystem currently covered by Defender on-access scans? Is this new verification feature an implementation of [SafetyNet](https://developer.android.com/training/safetynet/verify-apps) or [Play Integrity API](https://developer.android.com/google/play/integrity)? How will this show up in managed (e.g. Defender Endpoint) solutions? Will the [Microsoft Security Intelligence submission portal](https://www.microsoft.com/en-us/wdsi/filesubmission/) be updated to handle false positive reports for these packages? Any other docs?
